### ISSUE TITLE
Weed plants are disappearing in the THC garden.

### ISSUE THREAD

**Issue by CadillacKoopa** [echoroleplay/feedback#73]
Each garden has 100 plants at all times. We lose at least 15-37 plants per storm. They are watered and documented every time we work in the garden which is every 12 hours. This is happening at both THC EAST and THC WEST. Two different locations entirely. This began after the update to weed. 

**Reproduction Steps:** Clearly describe the steps to reproduce the bug. 
1. Step one - plant 100 plants
2. Step two - come back after the next storm passes to find at least 15% of the plants are gone.

**Expected Behavior:** Plants should remain when appropriately cared for

**Actual Behavior:** Some plants are dying out in less than 12 hours after being watered or planted

The screenshots show the missing plant spots and a few logs showing the amount we had to plant being more than what we harvested due to losses.
**Screenshot(s)/Video(s):** If possible, add screenshots or videos to help explain your problem. 
![Screenshot 2023-06-02 043630](https://github.com/echoroleplay/feedback/assets/76125478/44d06219-a1b5-4cbb-8cb7-aa2a228077d3)
![Screenshot 2023-06-02 042627](https://github.com/echoroleplay/feedback/assets/76125478/8e5faba0-397d-4511-bcec-a6a7b5605f10)
![Screenshot 2023-06-01 035812](https://github.com/echoroleplay/feedback/assets/76125478/2a5af1f4-b3e1-4e00-9bf7-23a1367492a4)
![Screenshot 2023-06-02 043807](https://github.com/echoroleplay/feedback/assets/76125478/8b88d1c1-c39f-4ac6-aac8-adaba3baf240)

**Comment by Spyder-Byte** [comment]
![FiveM_b2802_GTAProcess_VPYWZC58AL](https://github.com/echoroleplay/feedback/assets/79812238/c6bcf295-eb94-460a-b0f9-2bc3fcb6df2e)
To add to this - Planted, fertilized, watered, right before storm. Come back and plants are gone.

**Comment by CadillacKoopa** [comment]
So I believe I have found out the issue. Once planted, if you move a plant you are then unable to move it again. You get the prompt "attemped plant pickup failed". Every plant that I have gotten this error on has been gone at next restart. Thus, I am sure that something in the check of the plant that was changed, no longer recognizes the plant once moved. Therefore, the plant is wiped at restart as it doesn't register anymore. 


![Screenshot 2023-06-03 021208](https://github.com/echoroleplay/feedback/assets/76125478/eab0e8b8-eacb-4086-8a0f-02d1e932316f)
